src/documentUtil.js

```diff
@@ -61,6 +72,10 @@ var documentUtil = (function () {
         var selectorRegex = matchingSimpleSelectorsRegex(simpleSelectorList);
 
         asArray(element.querySelectorAll('style')).forEach(function (styleElement) {
+            if (typeof styleElement.sheet === 'undefined') {
```

Comment:
I've taken up the habit of commenting on the workarounds that go into the library. It hopefully reduces the amount of "WTF" folks will have when reading this later. Especially considering that this library feels like a one big workaround.
Feel free to do the same, possibly linking to our discussion on the issue tracker.